src/model/mod.rs

```diff
@@ -309,20 +346,39 @@ impl Model {
     ///
     /// ## Return
     /// Model: The Model created
-    pub fn new(stats: Stats, modifiers: Vec<Modifier>) -> Model {
-        Model { stats, modifiers }
+    pub fn new(stats: Stats, modifiers: Vec<modifier::Modifier>) -> Model {
+        let mut boosted_stats = stats.clone();
+        for modifier in &modifiers {
+            boosted_stats.apply_modifier(modifier);
+        }
+        Model {
+            stats,
+            boosted_stats,
+            modifiers,
+        }
     }
-    /// Get the Stats of the Model
+    /// Get the Stats of the Model taking account of the modifiers
     ///
     /// ## Return
-    /// Stats: The Stats of the Model
+    /// &Stats: The Stats of the Model
     pub fn get_stats(&self) -> &Stats {
```

Comment:
The name of the function isn't right it doesn't return the stats but the `boosted_stats`